src/test/java/org/springframework/hateoas/mediatype/PropertyUtilsTest.java

```diff
@@ -296,6 +320,20 @@ public void setFirstname(String firstname) {
 	static class JacksonCustomizations {
 		@JsonProperty("renamed") String property;
 	}
+	
+	@Value
+	@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+	static class JacksonPropertyNamingStrategyCustomizations {
+		@JsonProperty("small_property") String smallProperty;
+		String no_Name;
+	}
+	
+	@Value
+	@JsonNaming
+	static class JacksonNullPropertyNamingStrategyCustomizations {
```

Comment:
I would consider renaming this to `JacksonDefaultPropertyNamingStrategyCustomizations` for clarity.